sql/sql_expression.cc

```diff
@@ -1076,14 +1076,11 @@ const SqlExpression::Operator operators[] = {
     { "!<", false,       SqlExpression::compar, 4, "Not less than" },
     { ">", false,        SqlExpression::compar, 4, "Greater than" },
     { "<", false,        SqlExpression::compar, 4, "Less than" },
-    { "NOT", true,       SqlExpression::booln,  5, "Boolean not" },
+    { "NOT", true,       SqlExpression::booln,  5, "Unary not" },
     { "AND", false,      SqlExpression::booln,  6, "Boolean and" },
     { "OR", false,       SqlExpression::booln,  7, "Boolean or" },
     { "ALL", true,       SqlExpression::unimp,  7, "All true" },
     { "ANY", true,       SqlExpression::unimp,  7, "Any true" },
-    { "BETWEEN", false,  SqlExpression::unimp,  7, "Between operator" },
```

Comment:
They are actually implemented now, though not as explicit binary operators, so I took them out of the unimplemented list :)